### PR TITLE
Add M3 tripod to allow using M2HB anywhere

### PR DESCRIPTION
**Pull request by tornikeo**
#### Summary
Content "Add M3 tripod to allow using M2HB and Mk19 anywhere"

#### Purpose of change

To allow using M2HB and Mk19 anywhere, using a tripod. 

image

#### Describe the solution

Create M3 tripod - make it a default mod, since the M2 and Mk19 guns are useless without them. Make a flag for TRIPOD. Update the condition for terrain reqs to allow tripod use.

image

image

image

image

image

image

image

Who's laughing now? >:-)

#### Describe alternatives you've considered

Instead allow strength-check for condition (minigun mutant bear comes to mind).

#### Testing

See the images above.

#### Additional context

None

### CODE REVIEW

**Comment by Drew4484** [comment]
@anoobindisguise Does this overlap with your mounted gun work?

**Comment by tornikeo** [comment]
It's an underbarrel mod. Mounting doesn't matter. Can mount. Lemme check real quick.

**Comment by GuardianDll** [comment]
Can you also update a description of "you must stand blablabla" to mention you can prone?

**Comment by TheShadowFerret** [comment]
>make it a default mod, since the M2 and Mk19 guns are useless without them.

Standing on certain terrains such as windows make them usable tho 🤔

**Comment by NetSysFire** [comment]
>Can you also update a description of "you must stand blablabla" to mention you can prone?

Honestly, this should be a flag or something similar. This is out of scope for this PR though, but it should be done eventually. See #55821

File: data/json/items/gunmod/underbarrel.json [hunk 1]
```diff
@@ -20,6 +20,38 @@
...
+    "symbol": "A",
+    "color": "brown",
+    "location": "underbarrel",
+    "mod_targets": [ "m2browning", "mark19" ],
+    "handling_modifier": 23,
+    "min_skills": [ [ "weapon", 3 ] ],
+    "flags": [ "TRIPOD", "SLOW_WIELD" ]
+  },
+  {
+    "id": "m205_tripod",
```
Comment (BrettDong) [inline data/json/items/gunmod/underbarrel.json:47]: `"type"` is missing in this item definition.

Comment (tornikeo) [reply data/json/items/gunmod/underbarrel.json:47]: I thought "copy-from" already did that. Anyway, just added it.

**Comment by Night-Pryanik** [comment]
@tornikeo could you please resolve conflicts?

**Comment by tornikeo** [comment]
@Night-Pryanik thanks for point it out :D I've completely forgotten about this - also, terrain use for using M2 should be severly nerfed, no way in hell an M2 can be accurately used from a cupboard or a window without heavy duty bracers

**Comment by tornikeo** [comment]
I no longer wish to contribute to this project. Thanks for all the fish.

**Pull request closed without merging**